index.html

```diff
@@ -58,6 +58,207 @@ <h4>LET'S CONNECT</h4>
+            <article>
+                <h1>About Myself</h1>
+            </article>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            <h4>LET'S CONNECT</h4>
+            <ul>
+                <li>
+                    <a title="The link will take you to my facebook profile" href="https://www.facebook.com/Mpdaddy/">
+                    <i class="fa fa-facebook social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
+                        <i class="fa fa-twitter social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my instagram profile" href="https://www.instagram.com/mpdaddympd/">
+                        <i class="fa fa-instagram social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my linkin profile" href="https://www.linkedin.com/in/anold-mwawasi">
+                        <i class="fa fa-linkedin social"></i>
+                    </a>
+                </li>
+            </ul>
+            <div class="getresume">
+                <a title="get my resume" href="#">Get my resume</a>
+            </div>
+            <article id="what_i_do">
+                <div>
+                    <div class="headlang">
+                        <h5>Languages</h5>
+                        <i class="fa fa-angle-down fontarrows"></i>
+                    </div>
+                    <!--languages container-->
+                    <div class="langcontainer">
+                        <div class="langgrid">
+                            <img src="images/Ellipse 1.png" alt="">
+                            <p>javascript</p>
+                        </div>
+                        <div class="langgrid">
+                            <img src="images/Ellipse 1 (1).png" alt="">
+                            <p>HTML</p>
+                        </div>
+                        <div class="langgrid">
+                            <img src="images/Ellipse 1 (2).png" alt="">
+                            <p>CSS</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="headlang padd_bordder_bottom">
+                    <h5>Frameworks</h5>
+                    <i class="fa fa-angle-right fontarrows"></i>
+                </div>
+                <div  class="headlang padd_bordder_bottom">
+                    <h5>Skills</h5>
+                    <i class="fa fa-angle-right fontarrows"></i>
+                </div>
+            </article>
+        </section>
```

Comment:
Please remember to implement the border-top-right-radius as seen in Figma design as this does not only satisfy the requirement of making our design look exactly like the template but it is a nice little detail, isn't it? 😉 

Your design: 
![image](https://user-images.githubusercontent.com/107795045/236198166-a2eeaec4-525b-4f2f-8be8-e64dc607ab18.png)

Figma design: 
![image](https://user-images.githubusercontent.com/107795045/236198344-e7a9e2ea-f8dc-48f2-857b-3fe455b5b1aa.png)